verhoeven2021replication/openreview/content.tex

```diff
@@ -0,0 +1,305 @@
+
+Posterior collapse, a situation where the decoder network essentially ignores the encoder output, proved a serious problem for text generation. To overcome this issue, an aggressive training regiment was used \cite{he2019lagging}. Here, the encoder network is trained until convergence before updating the decoder network, resulting in stable and informative signals for the decoder network. This method of training is necessary for the first few epochs, but quickly ameliorates the situation and allows for regular VAE training to continue. However, given the sheer computational expense of using aggressive training, combining this with MC-estimation of $I(\alpha;Y)$ would quickly prove intractable. As such, both the classifier and generative model were first trained disjoint, resulting in a regular VAE text-generator, before attempting to fine-tune into a functioning GCE.
+
+
+\subsection{Implementation}
+
+Upon start of this project, the original authors' official code repository was non-functioning. Hence, all models were implemented from scratch using the descriptions and guidelines provided in the original paper. The only exception to this was the MC-estimation for information flow, although this was further optimised during the project. We exclusively used PyTorch, PyTorch-Text and PyTorch-Lightning for implementation, and all models were trained on Nvidia GeForce GTX 1080 GPUs provided by Surfsara's LISA cluster computing service. 
+
+
+%%%%%%%%%%%%%%%%%%%%%%at%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\section{Experimental Setup and Code}
+\label{sec:experiments}
+\begin{figure}[t]
+\centering
+    \begin{subfigure}[t]{.23\linewidth}
+        \includegraphics[width=.9\textwidth]{openreview/pictures/Figure3/alpha_1.png}
```

Comment:
The compilation issues are due to the incorrect file path. When you compile using `make` from the `journal` folder, the latex compiler should expect the relative path of `../openreview/pictures/Figure3/alpha_1.png`. Please fix all the relative paths and then run `make` from journal folder.

Reply:
Still not changed it seems